tests/FunctionalJavascript/AddCorporateFormTest.php

```diff
@@ -0,0 +1,174 @@
+<?php
```

Comment:
Let's move the tests inside a `/src` folder: `src/FunctionalJavscript/Class`

You will probably have to update the composer.json dev autoloader as well.

Reply:
Moved into `/tests/src` folder